wanaprez/prez.js

```diff
@@ -11,12 +11,26 @@ const KEY_LEFT = 37;
 const KEY_UP = 38;
 const KEY_RIGHT = 39;
 const KEY_DOWN = 40;
+const KEY_F = 70;
 
 let config = {
     url: "/",
     slide: 0,
 };
 
+function _toggleFullscreen() {
+	const elem = document.documentElement;
+	if (!document.fullscreenElement) {
+		elem.requestFullscreen().catch((err) => {
+			alert(
+				`Error attempting to enable full-screen mode: ${err.message} (${err.name})`
+			);
+		});
+	} else {
+		document.exitFullscreen();
+	}
+}
```

Comment:
Please use 4 spaces to indent the code, not tabs

```suggestion
function _toggleFullscreen() {
    const elem = document.documentElement;
    if (!document.fullscreenElement) {
        elem.requestFullscreen().catch((err) => {
            alert(
                `Error attempting to enable full-screen mode: ${err.message} (${err.name})`
            );
        });
    } else {
        document.exitFullscreen();
    }
}
```